hw/dv/tools/dvsim/common_sim_cfg.hjson

```diff
@@ -111,6 +111,7 @@
       name: smoke
       tests: []
       reseed: 1
+      run_opts: ["+smoke_test=1"]
```

Comment:
I will recommend adding a small comment here indicating what it does: 
```
// Knob used to configure an existing test / vseq to have a shorter runtime.
```

Reply:
Good point! Thanks Sri!